draft-shafranovich-rfc4180-bis.md

```diff
@@ -252,6 +252,16 @@ Example of a CSV file with empty lines:
    CRLF<br/>
    zzz,yyy,xxxCRLF
 
+However, if the records are only made up of one field it is not possible to
+differentiate between an empty line, and an empty and unquoted field. This
+differentiation might play an important role on database exports/imports.
```

Comment:
perhaps "an important role in some implementations such as database exports/imports"

Reply:
thanks